README.md

```diff
@@ -1 +1,253 @@
+   -  Find the ***sl_power_manager_hal_s0_s1.c*** in the path **gecko_sdk_3.2.1/platform/service/power_manager/src/sl_power_manager_hal_s0_s1.c** , and open it.
+   -  Try to modify the ***void sli_power_manager_handle_pre_deepsleep_operations(void)*** and ***void sli_power_manager_restore_high_freq_accuracy_clk(void)***, Simplicity Studio will pops up a warning box, click the **Make a Copy** to copy the this file from SDK library to project workspace.  
+      <img src="images/make a copy.png" width="50%" height="50%">
+
+   - Modify the functions in ***sl_power_manager_hal_s0_s1.c*** file as the follows.    
+
+      ```C  
+      /***************************************************************************//**
+      * Handle pre-deepsleep operations if any are necessary, like manually disabling
+      * oscillators, change clock settings, etc.
+      ******************************************************************************/
+      void sli_power_manager_handle_pre_deepsleep_operations(void)
+        {
+          if (is_hf_x_oscillator_used) {
+              CMU_OscillatorEnable(cmuOsc_HFRCO, true, true);
+              CMU_HFRCOBandSet(cmuHFRCOFreq_38M0Hz);
+              CMU_ClockSelectSet(cmuClock_HF, cmuSelect_HFRCO);
+              CMU_OscillatorEnable(cmuOsc_HFXO, false, false);
+              sl_board_disable_oscillator(SL_BOARD_OSCILLATOR_TCXO);
+          }
+        } 
+        
+        /***************************************************************************//**
+      * Handle post-sleep operations if any are necessary, like manually enabling
+      * oscillators, change clock settings, etc.
+      ******************************************************************************/
+      void sli_power_manager_restore_high_freq_accuracy_clk(void)
+      {
+        if (is_hf_x_oscillator_used) {
+            sl_board_enable_oscillator(SL_BOARD_OSCILLATOR_TCXO);
+        }
+      } 
+      ```
+
+   -  Include The TCXO control head file. Add the ***#include "sl_board_control.h"*** to this file. 
+   -  Add the ***sli_power_manager_private.h*** path to compiler. **Right click the project**->**properities**->**C/C++ build**->**setting**->**Tool setting**->**GNU ARM compiler**->**includes**, add the ***"${StudioSdkPath}/platform/service/power_manager/src"*** to path as the following figure.  
+      <img src="images/include path.png">
+
+8. **Calculate the minimum of Rx on time**
+   - Preamble detect need at least 40 bits. The formula is as follows.  
+***Trx = 40 / 9.6 = 4.167 ms***
+   - Find ***the sl_duty_cycle_config.h*** in **config** catalog, and Modify the macro **DUTY_CYCLE_ON_TIME** to 4167.
+      ```C
+      #define DUTY_CYCLE_ON_TIME      (4167)
+      ```   
+9. **Enable EM2 mode and disable button**  
+Find the ***the sl_duty_cycle_config.h*** file as above, and enable EM2 mode and disable button as following.
```

Comment:
- Open the ***sl_duty_cycle_config.h*** file in the config folder, enable EM2 mode and disable button as following.